apcd-cms/src/apps/admin_regis_table/templates/list_registrations.html

```diff
@@ -33,35 +40,7 @@ <h1>List Registrations</h1>
       {% endif %}
     </div>
   </div>
-  <table id="registrationTable" class="registration-table">
-      <thead>
-          <tr>
-              {% for k in header %}
-              <th>{{k}}</th>
-              {% endfor %}
-          </tr>
-      </thead>
-      <tbody>
-          {% for r in page %}
-              <tr>
-                  <td>{{r.biz_name}}</td>
-                  <td>{{r.year}}</td>
-                  <td>{{r.type}}</td>
-                  <td>{{r.location}}</td>
-                  <td>{{r.reg_status}}</td>
-                  <td>
-                    {% include "view_registration_modal.html" %}
-                    {% include "edit_registration_modal.html" %}
-                    <select id='actionsDropdown_{{r.reg_id}}' class='status-filter' onchange="openAction('{{r.reg_id}}')">
-                      <option value="">Select Action</option>
-                      <option value="viewRegistration">View Record</option>
-                      <option value="editRegistration">Edit Record</option>
-                    </select>
-                  </td>
-              </tr>
-          {% endfor %}
-      </tbody>
-  </table>
+  <div id="react-root"></div>
```

Comment:
Nice. Rather than implement Portal and mimic URLs, load the React app onto the existing pages. 👍